Projects/Restaurant/index.html

```diff
@@ -0,0 +1,262 @@
5l7qe9efbjseSazRap7NwXbdx1deCPuPUelUjKtMlOFnrgQwqP1/hKuK577DfiYl/ba1EW7vAPCOf8AKTwfQ/Ka6bp9UGHNU62iDXhlI8Z9kyQSBP71TNR4OysVII+IruDCo7xDwxLoyM9xzWtPsytdHCfFdDcRTg4yCOMd/Sqn4prr1t1KnECcSpOSY+vSune2/s1rmbYiM9mRLJ5iVkTKjOM9K534n4Xfs7Va2YndG3aeY4I7jr3FRlXI7cduH/DWg1LG3/iOSWO7mAvRhEEKfiI45rdzYXJLDY8Aj+XykwZnBwpn1+FLsaJmtMotncwE4kfm3QPjBx/YzXeH3vdkvYubCvKK0r5sT0IGfp16C/BRqlY9pkUsLg3ACFBwQ28kAZ6ZnnP0p7V6JVLLdkzG0iD5lMsJHAMx/c01bLe4UKQdyjqCV8wJ3IMTg9z8Ks3s57IXNS0ebaYaWDBNuDkkGScY9KV9jpqiFseVWIO4vbhCAJAcAEx+qTxxAjmpDSeFNe227en3ldu0wXYbeG7Lnv2Miuj6X8NdPAF5t4BDbVG1ZAgScsfrV00mlt2l220VB2UAftRWNvslL8iK0tnLPDPwwvXCf4htikYhtzk9zyB96m/Dvwl0afna4+Z5VQfiQJ+9dADUoNVaOV5ZMiPDPZnT6cRathevSZHBnmpEaNfX6mn91a301sTRpNOo4Ap0Cm/eUk3aBtD01otQ7X6ZfUUaBYS1ymHu0Nc1FC3NTTKIrYTcvULcv0Je1VCvdmmoATd1FCXLk0u3p2c4FS+h8I71m0g0R2l0RczVg0WgAouzpgtBePeP2NGm662Y8qDLt8B29TipymUjBthms1Vuxba5cYIiiWYmABXCvb728fWv7tAyadTKjrcI/ncfsvTk5417Ze1l7VuN522+Vtg+Vecn9Tev0ioHTaa5eYBE3dz/ACj51yTy3pHdjxcdvsGXUj1+lZVpteyLEAl1mP0z95E1lQ0XtnN7hPSaRvHrSyZ5Y/DFM3XkcnHyrsOI0zjvVv8AZX8RdRpIRyb1oYg/nUf5WJz8D9apRNb96f7Apk2hWk+z0x7M+3Om1a/4dwE9VOHHxU5+fFWq1fVhzXj5bzqQVJUjIZcMPgRkVcfZ/wDE/V6eFun36jv5XA/5hg/MfOnUk+yTxtdHpF7U0DrdCtwRcRXXsyg/vVO8A/E7S3gNz+7J6Pj/AOXH3q66TxS3cEhgZ7GmEaafoEs6e3bwttV+CgcU6boIgiRUjtU00+iBopoVpshtP4LpUuG6ti2HbkhR/wBvnUot0Ui5oD0NNNpnHrW0a35CffVv31BBG6yPgJpwIO/9KNI1hHv6z39M7F/s0rYtYAv+IrX8RTZRa1sX0oGFHU0htRW/KO32pQcdB9j/AKUTDRZj0Na92x9KfAY8Kf2pxdM57D70LDQH/D9zWjZUf71JJ4eerH5Yoi3olHShyQVFkINOWwq/MiKf03go5apqAB0FQHi/ttotPIa8rMP5bfnae2MD5mlcx1CyZs6RV6UnX+IWtOm+7cW2o6sQPp3PoK5R45+LF5yV01sWl/W8O8dwv5V+9VK22q1j7iXvMf53JgTjE4A9FqMsqR0QwPzov/tJ+KQgro1np71x/wDwn9W+lc+LarV3CwL3GY+ZzMdstx8hVj8I9jlUh7x3kZ2gQgjuBz05q0WrIUQqgARxjtXPLI5HTGMY9FS8O9kFAm8dx7fy9/iasmi0aJAVABwIjFElDmnFNIFs2oEcftWUgk961WAee/4Fx0NI/hG/Sft/rUsXPU/c/wCtI55J+9dVs56REGww/l/alC2/YfSjHVf1H60ztng/vRsFA7BusfSmrgPb7US4PGPrTQmiARpr7IcCR1FWXwrxa5b81m4yfAn7jg1W2U+tZZuMh3A/EdDWa9BTrs6n4Z+JWrtQLgS6PmrfUY+1Wnw38WrDQLiXEPwDD7Z+1cc0+qDjHPatuKCySXYXig+j0Povb7RXIi+gPZjtP0aKndP4tafKurfAg15cXsawqVypI9QSD9RTLIvKEeD0z1YLyHtW4U15RTxrW2s29Vfjt7xzHyJNG6f2+8SXjVufiEP/AE0/KInxSPUJsqela/hk7CvNtv8AE/xMf+cp+Ntf6U8PxX8T/Xa//X/9q3JewfGz0Z/Cp2FbGnTsK86H8V/Ezw9r5W//ALU2fxM8Ub/z1Hwtr/WtzXs3xM9Ii0o6ClSo7V5iu+3fiR51dz4AIv3C0Hf8c1l38+pvn/8AI4H0BpXNDLCz1Jf8QtIJd0UepA/eoPXe3/h1n82qtk9kO8/RJrzU1ovliWPcmT96ctWTMdKV5EUWD9nbvEPxj0i/8G1du/IIv/yM/aq5r/xZ1dz/AIVu3aB65dh9YH2rnRSBk/31ovwjw+9fbbZWR+oyF+vUVN5GyqwxRJeI+O6rUH/Gv3HHbdCT/wAogfak6HQXL3/DUn/Nwsevf4CrR4X7FqsG6d7dug9AvX5zVnXSoigAADioynZVJLSKn4J7Lp+a9525Kn8n061btNpgo6DsOKTbgcYxz9qd5+OKS7C2Oo2MD5n/AEpJf9hx9K0inFENbj0j/uKIgNtnp0raLAj5UzrddbtKXdgoHUmB96p/iHtsTK6e0zH9T+VfiByftTJNmbSLqyA1qudN4nqmyXcE/pZAPkCKym4MXminkZ+dF6ZBtOBwa3WVYiMhRHFC3eaysrBB2pLDBrdZTABWrTARWVlEAnSmLgjvU0vX4/1rKykmPAUvNLXj++9ZWUhQYuc1HOPOfjWqymQrFsKQ1ZWUQGWuTRZ6fD/WsrKDCh+0o7dqXZOa3WUjHRu9gGO4/ejAPLWVlKx0b8HQNqLYYAiGMHIkHFdY8DtgWxAAmeg7msrKXJ4DHokU5pu9/wBX/VWqypvoCGrPX4Gl2+B8RWVlFGYdbH9f3oXVMZPz/asrKIqOXe0F1m1BDEkAtAJJAz0B4pKOY5PSsrK6V0iL/wBBBY9zWVlZQMf/2Q=="
```

Comment:
why is the url so long?

Reply:
This url is of image used in website, the image was one of the stock images only.